docker-compose.dev.yml

```diff
@@ -0,0 +1,21 @@
+services:
+  app:
+    build:
+      context: .
+      target: dev
+    container_name: app-api
+    restart: unless-stopped
+    ports:
+      - "80:80"
+      - "443:443"
+    volumes:
+      - ./:/app
+      - caddy_data:/data
+      - caddy_config:/config
+      - runtime:/app/runtime
+    tty: true
+
+volumes:
+  caddy_data:
+  caddy_config:
+  runtime:
```

Comment:
I'd link it to app's runtime directory

Reply:
There's no app's runtime directory. In prod, no application source exists outside the container.